src/helics/apps/helicsWebServer.cpp

```diff
@@ -1003,30 +1020,72 @@ void WebServer::stopServer()
 
 void WebServer::mainLoop(std::shared_ptr<WebServer> keepAlive)
 {
-    if (http_enabled_) {
+    if (mHttpEnabled) {
         if (config->isMember("http")) {
             auto V = (*config)["http"];
-            helics::fileops::replaceIfMember(V, "interface", httpAddress_);
-            helics::fileops::replaceIfMember(V, "port", httpPort_);
+            helics::fileops::replaceIfMember(V, "interface", mHttpAddress);
+            helics::fileops::replaceIfMember(V, "port", mHttpPort);
+            bool external = helics::fileops::getOrDefault(V, "external", false);
+            helics::fileops::replaceIfMember(V, "all", external);
+            if (external) {
+                mInterfaceNetwork = static_cast<int>(gmlc::networking::InterfaceNetworks::ALL);
+            }
```

Comment:
It looks like the ipv4 or ipv6 options will override the external/all option -- should we move this block below the checks for ipv4 and ipv6 so that it will take precedence?

Reply:
probably